src/components/ProductList/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import ItemList from 'components/ItemList';
+
+function ProductList({ values }) {
+  return (
+    <>
+      {values && (
+        <div>
+          {values.docs.map((doc) => (
+            <div key={doc.id}>
+              <ItemList
+                itemName={doc.data().item}
+                docId={doc.id}
+                lastPurchasedDate={doc.data().lastPurchasedDate.toMillis()}
```

Comment:
I have a doubt about it, what will it happen if the product wasn't purchased before (e.g. Creating a new list and we add a new product)? 🤔 🤔  I believe you can move `toMillis` to your Item component or verify it before (It's only a suggestion, maybe there will be better solutions).

Reply:
good catch @alxmcr ! I'm also curious what `toMillis` does? I don't recognize it as a method of the Date object and I don't see it declared anywhere 🤔

Reply:
You're right. We already did that change. Thank you!